modules/calib/src/multiview_calibration.cpp

```diff
@@ -482,43 +506,55 @@ static void checkConnected (const std::vector<std::vector<bool>> &detection_mask
 }
 }
 
-//TODO: use Input/OutputArrays for imagePoints, imageSize(?), Ks, distortions
-double calibrateMultiview (InputArrayOfArrays objPoints, const std::vector<std::vector<Mat>> &imagePoints,
-        const std::vector<Size> &imageSize, InputArray detectionMask,
-        OutputArrayOfArrays Rs, OutputArrayOfArrays Ts, std::vector<Mat> &Ks, std::vector<Mat> &distortions,
-        OutputArrayOfArrays rvecs0, OutputArrayOfArrays tvecs0, InputArray isFisheye,
-        OutputArray perFrameErrors, OutputArray initializationPairs, bool useIntrinsicsGuess, InputArray flagsForIntrinsics) {
+//TODO: use Input/OutputArrays for imagePoints(?)
+double calibrateMultiview(
```

Comment:
Does this algorithm requires all images at once? If not, it would be more convenient (and efficient) to make iterative interface, e.g.:
```.cpp
MultiviewCalib calib(...);
for(...)
{
    calib.addFrame(...);
}
SomeResult res = calib.getResults();
```

Reply:
The algorithm does not use frames, but point coordinates only. The first step calibrates intrinsics and build spanning tree. The second step makes global optimization for all frames. So iterative API is not possible here.